faker/providers/date_time/__init__.py

```diff
@@ -1967,6 +1967,16 @@ def timezone(self):
         return self.generator.random.choice(
             self.random_element(self.countries)['timezones'])
 
+    def pytimezone(self, **kwargs):
+        """
+        Generate a random timezone object; taking the same arguments
+        as `timezone`.
+
+        :example tzfile('/usr/share/zoneinfo/Europe/London')
```

Comment:
Could we make this a little bit more user-friendly and have users pass timezones like `"Europe/London" instead of the full path?

Reply:
Sorry, I just tried to copy another - the usage simply looks like `faker.pytimezone()`, but the constructed result looks like that example. What should the example read?

The goal here is *not* to pass in a timezone, but to have faker generate a random timezone object.

Reply:
`self.timezone()` does not take any arguments, and so shouldn't `pytimezone`

The example should simply be `:example faker.pytimezone()`

Reply:
Thanks, I'll update the example. I tried to imitate the one below which was `Date('some str')` and seemed to be the `repr` of the returned object.

The idea with accepting kwargs is that if `self.timezone()` is modified for any future args (whatever they may be), no change is necessary here. I can't imagine `pytimezone` args would diverge from `timezone`'s, so to me it makes sense to tie the two together even if there's none today. Thoughts?

Reply:
>  I tried to imitate the one below 

Yeah, those are completely wrong and need to be fixed

> The idea with accepting kwargs is that if self.timezone() is modified for any future args (whatever they may be), no change is necessary here. 

As long as `kwargs` doesn't show up in the genecerated docs, I'm ok with having it. My priority is to not confuse the user.

Reply:
Thanks, I'll fix the example and check the generated docs.

Reply:
I've sorted the example, reworded the docstring to be less confusing than 'the same arguments', and updated the signature so that it matches others in the generated docs (particularly several in the internet provider).